cfme/tests/services/test_rest_services.py

```diff
@@ -97,6 +98,12 @@ def dialog(request, appliance):
     return _dialog(request, appliance)
 
 
+@pytest.fixture(scope="function")
+def service_dialogs(request, appliance, num=3):
```

Comment:
**QUESTION**
Why did you add `num` argument here?

Reply:
I could either add it inside the function as a variable or as a function argument. I chose argument because I use the same approach for other fixtures in this test file. On some places I call fixtures as normal functions, esp. when I need just single resource (``somefixture(..., num=1)``). This one is not used this way (yet), but since both possibilities to declare the variable are more-or-less equivalent, I haven't considered it as needlesly future-proofing :)